src/SiteConfig/ConfigBuilder.php

```diff
@@ -332,6 +350,24 @@ public function parseLines(array $lines)
                 continue;
             }
 
+            foreach (array_keys($this->extraConfigBuilders) as $prefix) {
+                if (strpos($command, $prefix . "_") === false) {
+                    continue;
+                }
+                $command = substr($command, strlen($prefix) + 1);
+                if (!isset($extraConfigItems[$prefix][$command])) {
+                    $extraConfigItems[$prefix][$command] = $val;
+                } elseif (!is_array($extraConfigItems[$prefix][$command])) {
+                    $extraConfigItems[$prefix][$command] = [
+                        $extraConfigItems[$prefix][$command],
+                        $val
+                    ];
+                } else {
+                    $extraConfigItems[$prefix][$command][] = $val;
+                }
+                continue 2;
```

Comment:
Nope. A single `continue` would continue from `foreach (array_keys($this->extraConfigBuilders) as $prefix)`. As we have found an extra config matching the prefix, we want to continue from the next line (`foreach ($lines as $line)`).